.ignore

```diff
@@ -0,0 +1,2 @@
+nix/stack.materialized
+nix/sources.nix
```

Comment:
There still seem to be changes in `sources.nix`?

Reply:
uhm... yes, weird. Let me check that.

Reply:
Fixed it.